src/main/scala/com/databricks/labs/validation/Validator.scala

```diff
@@ -48,7 +48,7 @@ class Validator(ruleSet: RuleSet, detailLvl: Int) extends SparkSessionWrapper {
           struct(
             lit(rule.ruleName).alias("ruleName"),
             (rule.inputColumn === rule.validExpr).alias("passed"),
-            lit(rule.inputColumnName).alias("permitted"),
+            rule.inputColumn.cast("string").alias("permitted"),
```

Comment:
Here I was expecting there to be the following columns.
ruleName - good
passed - good
logic - not present. This should be the SQL expr used to derive the "permitted"
permitted - allowed value or range
actual - the result of the expression value